src/api/posts/use-add-post.ts

```diff
@@ -1,17 +1,22 @@
 import type { AxiosError } from 'axios';
+import axios from 'axios';
 import { createMutation } from 'react-query-kit';
 
-import { client } from '../common';
+import { Env } from '@/core/env';
+
 import type { Post } from './types';
 
 type Variables = { title: string; body: string; userId: number };
 type Response = Post;
 
 export const useAddPost = createMutation<Response, Variables, AxiosError>({
   mutationFn: async (variables) =>
-    client({
-      url: 'posts/add',
+    axios({
+      url: `${Env.API_URL}posts/add`,
```

Comment:
Why not keeping the client call that already uses the baseURL?

Reply:
I reverted this change.